account_configuration/data/mail_template_data.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data noupdate="1">
+        <function name="write" model="ir.model.data">
+            <function name="search" model="ir.model.data">
+                <value eval="[('name', '=', 'email_template_edi_invoice'), ('module', '=', 'account')]"/>
+            </function>
+            <value eval="{'noupdate': False}"/>
+        </function>
+        <!--Email template -->
+        <record id="account.email_template_edi_invoice" model="mail.template">
+            <field name="body_html" type="html">
+<div style="margin: 0px; padding: 0px;">
+    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+        Dear
+        <t t-if="object.partner_id.parent_id">
+            <t t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
```

Comment:
@ByteMeAsap 
Commenting here, but it refers also to similar instances later in this file:

Why are you using `t-out` here? You could use `t-field`, and also get rid of the `or ''` statement.